generators/base/generator-base.mjs

```diff
@@ -59,6 +58,12 @@ import {
 import databaseData from '../sql-constants.mjs';
 import { CUSTOM_PRIORITIES } from './priorities.mjs';
 import { GENERATOR_BOOTSTRAP } from '../generator-list.mjs';
+import locateGenerator from './generator/blueprint-structure/generator-locator.mjs';
+import isBuiltInUserConfiguration from './generator/config/user.mjs';
+import isUsingBuiltInAuthorityConfiguration from './generator/config/authority.mjs';
```

Comment:
We should extract those function.

- these methods should be moved to `base-application` generator.
- I prefer to name the file with the function name if we are using `export default`.
- add an index api aggregator to reduce imported files.
```
import { isBuiltInUserConfiguration } from './generator/config/index.mjs';
```
```suggestion
import isBuiltInUserConfiguration from './generator/config/user.mjs';
import isUsingBuiltInAuthorityConfiguration from './generator/config/authority.mjs';
```

Reply:
The goal is to have multiple methods in the file, I'm at the very beginning.

An idea why the mts module isn't importable (javascript is far from being my specialty)?

Reply:
> The goal is to have multiple methods in the file, I'm at the very beginning.

Don't use export default then.
 
> An idea why the mts module isn't importable (javascript is far from being my specialty)?

Bug at loader.
https://github.com/jhipster/generator-jhipster/blob/0751d385f0da95e3efdbda2a352ab1a31ccbdb55/generators/base/index.mjs#L27-L28

Reply:
:'(